content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -14,7 +14,7 @@ jump_to:
     - Overview#overview
     - Supported channel parameters#supported-parameters
     - Using channel parameters with Ably libraries#using-parameters-ably
-    - Using channel parameters without Ably libraries#using-parameters-non-ably
+    - Using channel parameters outside of supported Ably libraries#using-parameters-non-ably
```

Comment:
I don't like describing these libraries as "non-supported", as if it's the library that's not supported. We ought to be able to say that the library does not support explicit channel params. Eg here it could be "Using channel params without Ably library support"